.github/scripts/setup_runners.sh

```diff
@@ -14,13 +14,25 @@
 #                $EC2_INSTANCE_NAMES: <names of instances> (required, sep ",")
 #                $GITHUB_TOKEN: GitHub PAT with repo scope and Admin Access to $GIT_URL_REPO
 #
-# Requirements:  The env that runs this script must have its AWS IAM with these configurations
-#                
-#                * SSM Role
-#                AmazonEC2RoleforSSM
-#                AmazonSSMManagedInstanceCore 
+# Requirements:  The env that runs this script must have its AWS resources with these configurations
+#
+#                1. Have an AWS user account with access to EC2 resource, remember the User ID
+#
+#                2. Create EC2 keypairs with name "odfe-release-runner"
+#
+#                3. Create EC2 Security Group with name "odfe-release-runner"
+#                   It has inbound rules of 22/9200/9600/5601 from Anywhere
```

Comment:
This needs to be locked down to only the IP ranges used by Github. Allowing access from anywhere is a security risk.

Reply:
@camerski I have tweaked the wording on this:
https://github.com/opendistro-for-elasticsearch/opendistro-build/pull/427/commits/7a3a6576096e44cad1fef3b3c1c9e57ca235910f